<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportAsImageDialog</class>
 <widget class="QDialog" name="ExportAsImageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export As Image</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Location</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Name:</string>
        </property>
        <property name="buddy">
         <cstring>fileNameEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="fileNameEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="browseButton">
        <property name="text">
         <string>&amp;Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="visibleLayersOnly">
        <property name="text">
         <string>Only include &amp;visible layers</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="currentZoomLevel">
        <property name="text">
         <string>Use current &amp;zoom level</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="drawTileGrid">
        <property name="text">
         <string>&amp;Draw tile grid</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="includeBackgroundColor">
        <property name="text">
         <string>&amp;Include background color</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="splitIntoTiles">
        <property name="toolTip">
         <string>Splits the image into tiles which are saved as individual files using a zero-based numerical suffix</string>
        </property>
        <property name="text">
         <string>&amp;Split into tiles (overwrites existing files)</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QFormLayout" name="tileSizeLayout">
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="tileWidthLabel">
            <property name="text">
             <string>Tile width</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="tileWidthSpinBox">
            <property name="toolTip">
             <string>The width of each tile</string>
            </property>
            <property name="minimum">
             <number>8</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="value">
             <number>512</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="tileHeightLabel">
            <property name="text">
             <string>Tile height</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="tileHeightSpinBox">
            <property name="toolTip">
             <string>The height of each tile</string>
            </property>
            <property name="minimum">
             <number>8</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="value">
             <number>512</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExportAsImageDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExportAsImageDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
